homeworks/react-continued-3/yana_yahello/src/LecturesDetails.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {useState} from 'react';
+import PropTypes from 'prop-types';
+
+function LectureDetails({date, title, lecturer, link}) {
+    LectureDetails.propTypes = {
+        date: PropTypes.string,
+        title: PropTypes.string,
+        lecturer: PropTypes.string,
+        link: PropTypes.string
+    };
+
+    const [isOpened, setState] = useState(false);
+
+    return (
+        <div>
+            <h2 onClick={() => setState(!isOpened)}>Date: {date}</h2>
```

Comment:
I mean, pass that function to setState. 
Also, name `setState` is not clear, it may confuse

Reply:
Thanks, done!